recipes/constructive_geometries/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "constructive_geometries" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/constructive_geometries-{{ version }}.tar.gz
+  sha256: a52602689a941ee4407f73149899dbe6ba6ff935d74f1e59a66c79e7a11bb66f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - country_converter
+    - python >=3.6
+    - wrapt
+
+test:
+  imports:
+    - constructive_geometries
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/cmutel/constructive_geometries
+  summary: Simple tools to define world locations from a set of topological faces and set algebra
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - cmutel
```

Comment:
@cmutel, please confirm you're happy to be a maintainer here, too.

Reply:
Indeed :)